css/base/variables.css

```diff
@@ -390,4 +390,7 @@ body {
 
   /* Leaflet Mapping */
   --leaflet-map-outline-stroke-colour: #38f;
+  
+  /* Service Landing Page */
+  --service-cta-border: var(--border-width);
```

Comment:
I tihnk this should evaluate to `--border`, shouldn't it?

Reply:
Indeed. I missed the declaration of that in amongst the others. 

Any issue if I separate those meta variables out into their own section to make them more obvious?

Reply:
Yes, work away, happy to look at a PR for that to see how it works.

Reply:
It's only a small tweak, have included it in this PR [9ef8f6c](https://github.com/localgovdrupal/localgov_base/pull/200/commits/9ef8f6cd9b68aee15e11963eff159d97d82ed10d)